.github/workflows/cleanup-branch-builds.yml

```diff
@@ -9,17 +9,31 @@ on:
       - 'dependabot-**'
       - 'feat/**'
 jobs:
-  delete-package:
-    name: Delete Github Package for Branch
+
+  delete-dat-package:
+    name: Delete DAT-SNAPSHOT Github Package for Branch
     runs-on: ubuntu-22.04
+
     steps:
+
+    - name: Extract branch name
```

Comment:
can we consolidate branch name extraction?
` extract-branch-info:
    name: Extract Branch Info
    runs-on: ubuntu-22.04
    outputs:
      branch_name: ${{ steps.get_branch_name.outputs.branch_name }}
  
     - name: Extract branch name
        id: get_branch_name
        shell: bash
        run: |
          BRANCH_NAME=${GITHUB_REF#refs/heads/}
          echo "Branch name is $BRANCH_NAME"
          echo "BRANCH_NAME=$BRANCH_NAME" >> $GITHUB_ENV`

and then reference it as `version: "${{ needs.extract-branch-info.outputs.branch_name }}`

Reply:
true! thanks!